pkg/util/cobra.go

```diff
@@ -0,0 +1,20 @@
+package util
+
+import (
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// GetConfigPath gets the config path from a cobra cmd
+func GetConfigPath(cmd *cobra.Command) (string, error) {
```

Comment:
Probably fine for now but we should maybe consider using [viper](https://github.com/spf13/viper) for this.

Reply:
yeah, i have another pr locally where i'm doing this